### ISSUE TITLE
Cognitive regression

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7547]
**Preferred term label:**
Cognitive regression

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

617951

PMID: 29576217  4/4
**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added as HP_0034332
